contracts/hardhat.config.js

```diff
@@ -227,6 +243,7 @@ module.exports = {
       mainnet: MAINNET_GOVERNOR,
       holesky: HOLESKY_DEPLOYER, // on Holesky the deployer is also the governor
       base: BASE_GOVERNOR,
+      sonic: MAINNET_GOVERNOR,
```

Comment:
nit: We don't have a governor yet, so let's set it to index 1 for now

Reply:
Or deployer address

Reply:
yup thanks changed